v2/init/dp-static-file-publisher/service.patch

```diff
@@ -0,0 +1,29 @@
+diff --git service/service.go service/service.go
+index 45d58d5..35733fe 100644
+--- service/service.go
++++ service/service.go
+@@ -230,15 +230,15 @@ func (svc *Service) registerCheckers(ctx context.Context) (err error) {
+ 		log.Error(ctx, "failed to add KafkaFilePublishedConsumer checker", err)
+ 	}
+ 
+-	if err = svc.HealthCheck.AddCheck("S3 Public", svc.S3Public.Checker); err != nil {
+-		hasErrors = true
+-		log.Error(ctx, "failed to add public s3 client checker", err)
+-	}
+-
+-	if err = svc.HealthCheck.AddCheck("S3 Private", svc.S3Private.Checker); err != nil {
+-		hasErrors = true
+-		log.Error(ctx, "failed to add private s3 client checker", err)
+-	}
++	// if err = svc.HealthCheck.AddCheck("S3 Public", svc.S3Public.Checker); err != nil {
++	// hasErrors = true
++	// log.Error(ctx, "failed to add public s3 client checker", err)
++	// }
++
++	// if err = svc.HealthCheck.AddCheck("S3 Private", svc.S3Private.Checker); err != nil {
++	// hasErrors = true
++	// log.Error(ctx, "failed to add private s3 client checker", err)
++	// }
```

Comment:
Please remove the commented out code here too.

Reply:
as above

Reply:
👍